TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/ItemsListFragment.kt

```diff
@@ -50,42 +49,73 @@ class ItemsListFragment : Fragment(), AdapterView.OnItemClickListener {
         }
 
         view.findViewById<FloatingActionButton>(R.id.add_fab).setOnClickListener {
-            addItem()
+            DataProvider.createNote()
+
+            // Set selected item to newly created note (first element in list)
+            setSelectedItem(0)
+            startNoteFragment(selectedItemPosition)
         }
 
         return view
     }
 
-    private fun addItem() { }
-
     private fun setSelectedItem(position: Int) {
         listView?.setItemChecked(position, true)
+        selectedItemPosition = position
     }
 
     override fun onItemClick(adapterView: AdapterView<*>, item: View, position: Int, rowId: Long) {
-        val movieMock = arrayAdapter?.getItem(position)
+        startNoteFragment(position)
+    }
+
+    private fun startNoteFragment(position: Int) {
+        val note = arrayAdapter?.getItem(position)
         setSelectedItem(position)
```

Comment:
is it necessary to set the selected item here? it was just done in `onCreateView`